Update index.md (#56)
uk-tre/website: docs/structure/citizen-agency.md

# Citizen agency

## Charter

> This is the main document to refer to, it should contain all of the below and further detail: [citizen agency charter](https://www.dropbox.com/scl/fo/6knltj4t4l2vz32tun8y7/ACX3kkKF-W8yVSCWBFGFzEA?rlkey=wfsfix0jl4rhj8ovk0g6rhvwm&dl=0) 

Author: ```suggestion
This is the main document to refer to, it should contain all of the below and further detail: [citizen agency charter](https://www.dropbox.com/scl/fo/6knltj4t4l2vz32tun8y7/ACX3kkKF-W8yVSCWBFGFzEA?rlkey=wfsfix0jl4rhj8ovk0g6rhvwm&e=1&st=98f0i8na&dl=0) 
```

> This is the main document to refer to, it should contain all of the below and further detail: [citizen agency charter](https://www.dropbox.com/scl/fo/6knltj4t4l2vz32tun8y7/ACX3kkKF-W8yVSCWBFGFzEA?rlkey=wfsfix0jl4rhj8ovk0g6rhvwm&e=1&st=98f0i8na&dl=0) 

Author: ```suggestion
This is the main document to refer to, it should contain all of the below and further detail: you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing).

(sorry for the lack of a direct link, but it lives in dropbox)
```

## Overview

The Citizen Agency Working group is focused on documenting and outlining possible roles for the citizen within the data research arena, with specific focus on the Secure Data Environment / Trusted Research Environment facilities.  

This Working Groups was set up to support delivery of these overall macro concerns and objectives.

- To minimise Time-to-research and delay
- To make combining datasets easy and reduce barriers to research project scope
- To include Citizen’s agency into the “TRE landscape” and Information Governance. 

## Co-chairs and contact

**Chair:**  This is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk 
**Contact:**  This is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk

## How to get involved
(mailing list, slack channel, repositories…)

Mailing list: WG-CITIZEN-AGENCY-UKTRE-COMM@JISCMAIL.AC.UK
- Sign up form https://www.jiscmail.ac.uk/cgi-bin/webadmin?SUBED1=WG-CITIZEN-AGENCY-UKTRE-COMM&A=1 

Slack: #wg-citizen-agency channel on the [UK TRE Community slack](https://join.slack.com/t/uktrecommunity/shared_invite/zt-2gep86apc-QMLyIdrC2oIIsxTRzLxUqA)

> [Dropbox](https://www.dropbox.com/scl/fo/oqxpmnriszxr6q67ndkc5/ANwHMrobcot_ZxxpzbxYABs?rlkey=cut8e94siwdlcf3yqihscvvbf&dl=0)  

Author: ```suggestion
Dropbox: ask for the link to existing members or Pete Barnsley, you can also find it within this you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing). 
```

# Citizen agency

## Charter

This is the main document to refer to, it should contain all of the below and further detail: you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing).

(sorry for the lack of a direct link, but it lives in dropbox)

## Overview

The Citizen Agency Working group is focused on documenting and outlining possible roles for the citizen within the data research arena,

> with specific focus on the Secure Data Environment/Trusted Research Environment facilities. 

Author: ```suggestion
with specific focus on the Secure Data Environment/Trusted Research Environment facilities.
```

# Citizen agency

## Charter

This is the main document to refer to, it should contain all of the below and further detail: you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing).

(sorry for the lack of a direct link, but it lives in dropbox)

## Overview

The Citizen Agency Working group is focused on documenting and outlining possible roles for the citizen within the data research arena,

> with specific focus on the Secure Data Environment/Trusted Research Environment facilities.  

Author: ```suggestion
with specific focus on the Secure Data Environment/Trusted Research Environment facilities. 
```

This Working Groups was set up to support delivery of these overall macro concerns and objectives.

- To minimise Time-to-research and delay
- To make combining datasets easy and reduce barriers to research project scope
- To include Citizen’s agency into the “TRE landscape” and Information Governance

## Co-chairs and contact

> **Chair:** This is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk 

Author: ```suggestion
**Chair:** This is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk
```

> **Contact:** This is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk

Author: ```suggestion
**Contact:** This is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk
```

## How to get involved

(mailing list, slack channel, repositories…)

Mailing list: WG-CITIZEN-AGENCY-UKTRE-COMM@JISCMAIL.AC.UK

> - Sign up form https://www.jiscmail.ac.uk/cgi-bin/webadmin?SUBED1=WG-CITIZEN-AGENCY-UKTRE-COMM&A=1 

Author: ```suggestion
- Sign up form https://www.jiscmail.ac.uk/cgi-bin/webadmin?SUBED1=WG-CITIZEN-AGENCY-UKTRE-COMM&A=1
```

Slack: #wg-citizen-agency channel on the [UK TRE Community slack](https://join.slack.com/t/uktrecommunity/shared_invite/zt-2gep86apc-QMLyIdrC2oIIsxTRzLxUqA)

Dropbox: ask for the link to existing members or Pete Barnsley,

> you can also find it within this you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing). 

Author: ```suggestion
you can also find it within this you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing).
```

People interested in supporting this work should contact the primary contact or chair, or just add to the slack channel discussions.
The prime contact is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk

> 
> 

Author: ```suggestion
```
